Fixes(#61): Adds backend code coverage reference (#70)
oppia/oppia-web-developer-docs: Debug-backend-tests.md

**Make sure that you can reproduce the problem you are trying to debug when you run the test in isolation!** While rare, it is possible for a test failure to have been caused by previous tests. In these cases, you may not be able to reproduce the problem when you only run the test that initially failed.

## Complete coverage

> For identifying the missing coverage lines run the below command.

Reviewer: ```suggestion
To identify the missing coverage lines, run the below command.
```

Author: Updated

> For identifying the missing coverage lines run the below command.

Reviewer: ```suggestion
To identify the missing coverage lines, run the below command.
```

> For identifying the missing coverage lines run the below command.

Reviewer: ```suggestion
To identify the missing coverage lines, run the below command.
```

```console
python -m scripts.run_backend_tests --generate_coverage_report
```

> For more understanding about the backend code coverage kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).

Reviewer: ```suggestion
For more information about the backend code coverage, kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).
```

Author: Updated

> For more understanding about the backend code coverage kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).

Reviewer: ```suggestion
For more information about the backend code coverage, kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).
```

> For more understanding about the backend code coverage kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).

Reviewer: ```suggestion
For more information about the backend code coverage, kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).
```

> For more understanding about the backend code coverage kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).

Reviewer: ```suggestion
For more information about the backend code coverage, kindly refer [here](https://github.com/oppia/oppia/wiki/Backend-tests#coverage-reports).
```
